legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -165,3 +167,23 @@ def validate_continuation_in_court_order(filing: dict, filing_type) -> list:
         if err:
             return err
     return []
+
+
+def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
+    """Validate continuation EXPRO business by making a call to Colin API."""
+    msg = []
+    business_identifier_path = f'/filing/{filing_type}/business/identifier'
+    business_legal_name_path = f'/filing/{filing_type}/business/legalName'
+
+    if filing_json['filing'][filing_type].get('business'):
+        identifier = filing_json['filing'][filing_type]['business']['identifier']
+        legal_name = filing_json['filing'][filing_type]['business']['legalName']
+        response = colin.query_business(identifier)
+        response_json = response.json()
+        if response.status_code != HTTPStatus.OK:
+            msg.append({'error': 'Could not fetch business data for company from Colin.',
+                        'path': business_identifier_path})
+        elif legal_name != response_json['business']['legalName']:
+            msg.append({'error': 'Legal name does not match with company legal name from Colin.',
+                        'path': business_legal_name_path})
+    return msg
```

Comment:
Yeah, I guess you don't have to verify much here, only that the business exists.

Here's what the UI code does with the COLIN business data: https://github.com/bcgov/business-create-ui/blob/508671b0aaac1cad25ff1c6b253a8bf283cbc4ea/src/components/ContinuationIn/ExtraproRegistration.vue#L531

Reply:
Yes